File: src/sdk-reference/php/3/memory-storage/bitop/snippets/bitop-1.php

<?php

use \Kuzzle\Kuzzle;


$kuzzle = new Kuzzle('localhost');

try {
  $length = $kuzzle->memoryStorage()->bitop('key', 'AND', ['srckey1', 'srckey2', '...']);
}
catch (ErrorException $e) {

}

File: src/sdk/java/2/core-classes/memory-storage/bitop/snippets/bitop-1.java

String[] sourceKeys = {"srckey1", "srckey2", "..."};

kuzzle.memoryStorage.bitop("key", "AND", sourceKeys, new ResponseListener<Long>() {
  @Override
  public void onSuccess(int length) {
    // callback called once the action has completed
  }

  @Override
  public void onError(JSONObject error) {
  }
});
